main/settings/src/main/scala/sbt/Reference.scala

```diff
@@ -78,7 +78,7 @@ object Reference {
       case LocalRootProject    => "{<this>}<root>"
       case LocalProject(id)    => "{<this>}" + id
       case RootProject(uri)    => "{" + uri + " }<root>"
-      case ProjectRef(uri, id) => "{" + uri + "}" + id
+      case ProjectRef(uri, id) => s"""ProjectRef(uri("$uri"),"$id")"""
```

Comment:
> Remember that this output is shown for every `update`.

Perhaps `update` should use `Def.displayRelative` or similar function, that can display `root` as opposed to the full `ProjectRef`.